### ISSUE TITLE
Learner view: in a channel, independent practice quizzes can be taken, submitted, and the report can be seen

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#8509]
### User stories for this issue:

> As a learner, I want to be able to find an independent practice quiz in a channel so that I can take it, submit it, earn points, and view the report.

|Frame|Explanation|
|--|--|
|![Screen Shot 2021-10-20 at 8 53 50 PM](https://user-images.githubusercontent.com/13563002/138208943-7656359f-1d54-4605-9bcc-2fb03dbe83d2.png)|From channels tab, learner will go into a folder (see Figma link below for details)|
|![Screen Shot 2021-10-20 at 8 54 17 PM](https://user-images.githubusercontent.com/13563002/138208947-fe4542ad-0e4f-431a-97e0-e37ec754fa5a.png)|Find the independent practice quiz|
|![Screen Shot 2021-10-20 at 8 54 28 PM](https://user-images.githubusercontent.com/13563002/138208949-edfab565-3fed-46e6-a158-3abce165e009.png)|Engage with and submit quiz|
|![Screen Shot 2021-10-25 at 10.20.35 PM.png](https://images.zenhubusercontent.com/6050ea86e1747f15de7d8759/693a0d71-28d0-430e-a391-04eaad066aaa)| Modal should appear|
|![Screen Shot 2021-10-20 at 8 55 22 PM](https://user-images.githubusercontent.com/13563002/138208952-e18c9a49-fdc9-4668-99e6-b93ce788bf8f.png)|Learner earns points, completion is updated, and learner is redirected back to the folder|
|![Screen Shot 2021-10-20 at 8 25 03 PM](https://user-images.githubusercontent.com/13563002/138209139-ce6ea74e-2448-4056-bffa-c65dda5147c6.png)|Learner sees their quiz report if they click on the independent practice quiz|

### Notes and changes to current designs:
- [Figma for finding an independent quiz in a channel](https://www.figma.com/file/TfaSTT9gs6fPKhGdNUjffU/Channel-quizzes?node-id=7%3A2)
- There are three shared components that have already been created for use, `PracticeQuizRenderer`, `ReportsPracticeQuizLearnerPage`, `PracticeQuizTimer`
- All strings required should already be in `commonCoreStrings`, `commonCoachStrings`, or `commonLearnStrings`
- Routing will need to be created

### Criteria for success:
**Please copy these into your PR for "Steps for Testing"!**

This issue is done when the checkboxes and Gherkin stories below work
- [ ] Learner earns points after submitting the independent practice quiz and returning back to the lesson resource page 
- [ ] Learners can discover independent practice quizzes in Learn in the channels tab
```
Feature: Learners can discover independent practice quizzes in Learn
	Scenario: independent practice quizzes appear in the channels tab
		Given that I am on the channels tab
		When I navigate to the node with an independent practice quiz
			And I click on the independent practice quiz
		Then I can engage with the independent practice quiz
```
- [ ] Learners can see time information while taking an independent practice quiz in lesson or channel
```
Feature: Learners can see time information while taking an independent practice quiz in lesson or channel	
	Scenario: Learners see how long they are taking on the independent practice quiz
		Given that I have entered the independent practice quiz page
		When the I start on the independent practice quiz
		Then I can see a time displayed as 'Time spent' above the question list
			And I can see that time increment by minutes

	Scenario: Learners see the suggested time to take on the independent practice quiz
		Given that I have entered the independent practice quiz page
		When the I start on the independent practice quiz
		Then I see a time displayed as 'Quiz duration' above the question list
			And I do not see that time increment
```
- [ ] Learners can see independent practice quiz report from independent practice quiz
```
Feature: Learners can see independent practice quiz reports from independent practice quizzes assigned in lessons or in channels
	Scenario: learner views independent practice quiz report after finishing the independent practice quiz
		Given that I have finished and submitted the independent practice quiz
			And I am back on in the channels tab
			And I see my score on the same independent practice quiz content card
		When I click on the content card
		Then I see my independent practice quiz report
			And I see my score and time information
			And I see a button to retake the independent practice quiz
```

**Comment by bjester** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Kolibri-Ecosystem-6047e525b8b9d900182e2d6f/issues/learningequality/kolibri/8509?planning-poker) with ZenHub @jredrejo @marcellamaki @sairina @rtibbles @MisRob @nucleogenesis @jamalex

**Comment by rtibbles** [comment]
Fixed in #8645
